addons/l10n_ec/data/account_tax_tag_data.xml

```diff
@@ -0,0 +1,695 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+	<data>
+		<record id="tax_tag_303" model="account.account.tag">
+			<field name="name">303</field>
+			<field name="applicability">taxes</field>
+			<field name="country_id" ref="base.ec"/>
+		</record>
```

Comment:
The real tax report is an excel file, and those codes represent the cells to fill. This excel file is standardized by the tax authority, and subject to changes quite often.

Reply:
An Excel file in the sense that it has multiple columns per row?  Because we could create a tax report and then put a button on it with "Export XLSX" and then customize that export.

Reply:
@Stentor double check if we can use the tax reports. The tax codes can be taken directly from these reports.